src/js/button.js

```diff
@@ -1,3 +1,8 @@
+import Component from './component';
+import * as Lib from './lib';
```

Comment:
Right, that assumes we've decided to drop everything on the main object, which I don't know that we've officially landed on that, and I feel like I've lost references to a few of the specific conversations we had around that.